File: Class_member/3.rb
class Cs
    @@count = 0
    def initialize()
        #@count 인스턴스에 속한 변수
        @@count = @@count + 1 #클래스에 속한 변수
    end
    def Cs.getCount() #밑에서 클래스로 호출하고 있으므로 Cs. 을 통해서 클래스로 만들어줌
        return @@count
    end
end

i1 = Cs.new()
i2 = Cs.new()
i3 = Cs.new()

p(Cs.getCount())

File: Class_member/3.py
class Cs :
    count = 0 #루비와는 달리 파이썬에서는 그냥 count로 선언 하면 된다.
    def __init__(self) :
        Cs.count = Cs.count + 1
    @classmethod
    def getCount(cls) :
        return Cs.count

i1 = Cs()
i2 = Cs()
i3 = Cs()
i4 = Cs()

print(Cs.getCount())